.gitlab-ci.yml

```diff
@@ -76,27 +73,25 @@ pytest:
   script:
     - python -V
     # Command to test our application
-    - pytest -v
+    - pytest -vvv
 
 
 # Build
 
 build:
   image: docker:19.03.12
   stage: build
-  needs: ["pytest"]
+#  needs: ["pytest"]
   services:
-    - docker:19.03.12-dind
+    - docker:dind
   variables:
     IMAGE_TAG: $CI_REGISTRY_IMAGE:$CI_COMMIT_REF_SLUG
+  before_script:
+    - echo $CI_REGISTRY_TOKEN | docker login -u $CI_REGISTRY_USER $CI_REGISTRY --password-stdin
```

Comment:
Why not use the dependency proxy here either?

Reply:
Because we are login in to our GitLab e.g `docker login dockerhub.ebi.ac.uk` using `CI_REGISTRY` (contains the address of the GitLab Container Registry) as you mentioned (I'm writing it down here for future reference 😃 )